common.lic

```diff
@@ -757,6 +757,11 @@ module DRC
     DRC.bput("stance set #{stance}", /Setting your/)
   end
 
+  def atmo(text)
+    string = "<pushStream id=\"atmospherics\"/>" + text
+    _respond(string, "<popStream id=\"atmospherics\" /><prompt time =\"#{Time.now.to_i}\">&gt;</prompt>")
```

Comment:
🧹 My stylistic suggestion is to format like this and avoid creating the `string` variable
```ruby
def atmo(text)
  _respond(
    "<pushStream id=\"atmospherics\"/>" + text,
    "<popStream id=\"atmospherics\" /><prompt time=\"#{Time.now.to_i}\">&gt;</prompt>"
  )
end
```